gradle/libs.versions.toml

```diff
@@ -17,6 +17,7 @@ clikt = "3.5.2"
 commonsCompress = "1.23.0"
 cvssCalculator = "1.4.1"
 cyclonedx = "7.3.2"
+diffUtils = "4.12"
```

Comment:
Does the "click to see diff" feature in IntelliJ still work with this new matcher?